include/ensmallen_bits/sgd/update_policies/quasi_hyperbolic_update.hpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file quasi_hyperbolic_update.hpp
+ * @author Niteya Shah
+ *
+ * Quasi Hyperbolic Update for Stochastic Gradient Descent.
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef ENSMALLEN_SGD_QH_UPDATE_HPP
+#define ENSMALLEN_SGD_QH_UPDATE_HPP
+
+namespace ens {
+
+/**
+ * Quasi Hyperbolic Update policy for Stochastic Gradient Descent (QHSGD).
+ *
+ * QHSGD is a update policy that is defined to be a parent of 
+ * most update policies(most update policies can reconstructed from
+ * QHSGD). This allows this method to combine the features of many optimisers
+ * and provide better optimisation control.
+ *
+ * @code
+ * @inproceedings{ma2019qh,
+ *   title={Quasi-hyperbolic momentum and Adam for deep learning},
+ *   author={Jerry Ma and Denis Yarats},
+ *   booktitle={International Conference on Learning Representations},
+ *   year={2019}
+ * }
+ * @endcode
+ *
+ */
+class QHUpdate
+{
+ public:
+  /**
+   * Construct the Quasi Hyperbolic update policy with the given parameters.
+   * @param v The quasi hyperbolic term.
```

Comment:
Can you add another empty line between the method description and the beginning of the parameter description.